exercises/concept/top-scorers/.meta/Exemplar.elm

```diff
@@ -0,0 +1,59 @@
+module AggregateScorers exposing (..)
+
+import Dict exposing (..)
+
+-- should mostly be doing the higher level things, like map, and less often doing
+-- the low level things, like get, delete
+-- todo: mention this in the instructions
+
+
+type PlayerName = String
+
+
+aggregateScorers : List PlayerName -> Dict PlayerName Int
+aggregateScorers playerNames =
+    Debug.todo "Use List.foldl and updateGoalCountForPlayer to convert the list in to a Dict"
+    --List.foldl updateGoalCountForPlayer Dict.empty playerNames
+
+
+updateGoalCountForPlayer : PlayerName -> Dict PlayerName Int -> Dict PlayerName Int
+updateGoalCountForPlayer playerName playerGoalCounts =
+    Debug.todo "implement updateGoalCountForPlayer function, to initialise or increment the goalcount for PlayerName"
+    -- let
+    --     updatePlayer existingPlayer =
+    --         case existingPlayer of
+    --             Just player ->
+    --                 { player | goals = player.goals + 1 }
```

Comment:
The types don't match here, you are returning a record rather than an `Int`, and also `updatePlayer` should be of type `Maybe v -> Maybe v`.

Reply:
Yep, I haven't tried to run this code or anything, I'll fix that up later!